medical_pharmacy/models/medical_pharmacist.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+from odoo.modules import get_module_resource
+
+
+class MedicalPharmacist(models.Model):
+    _name = 'medical.pharmacist'
+    _description = 'Medical Pharmacist'
+    _inherit = 'medical.abstract.entity'
+
+    @api.model
+    def _create_vals(self, vals):
+        vals.update({
+            'is_company': False,
+            'customer': False,
+        })
+        return super(MedicalPharmacist, self)._create_vals(vals)
+
+    @api.model
+    def _get_default_image(self, vals):
+        res = super(MedicalPharmacist, self)._get_default_image(vals)
```

Comment:
Oh on the note of avatars, if you have any medical icons laying around with use rights - we need them!